scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -1764,8 +1764,8 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
       case t: Name => Some(t)
       case t: Quasi => Some(t.become[Term.Name])
       case t: Term.Select => Some(t.name)
-      case t: Term.Placeholder => Some(copyPos(t)(Name.Anonymous()))
-      case t: Term.Eta => Some(atPos(t.endTokenPos)(Name.Anonymous()))
+      case t: Term.Placeholder => Some(copyPos(t)(Name.Placeholder()))
```

Comment:
Let's release it under 4.7, since it's an API addition and can change what people expect from the parser.

Reply:
will do, thanks.